manim/mobject/geometry.py

```diff
@@ -402,9 +387,25 @@ def __init__(self, point=ORIGIN, **kwargs):
 
 
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    CONFIG = {"radius": DEFAULT_SMALL_DOT_RADIUS}
+
+
+class AnnotationDot(VMobject):
+    def __init__(self, color=BLUE):
+        VMobject.__init__(self)
+        sourunding_dot = Dot().scale(1.3).set_fill(color=WHITE).set_z_index(-1)
+        innerdot = Dot().set_color(color)
+        annotation_dot = VGroup(sourunding_dot, innerdot)
+        self.add(annotation_dot)
+
+
+class LabeledDot(Mobject):
```

Comment:
```suggestion
class LabelledDot(Mobject):
```

Reply:
why the double l?
it think it is labeled